frontend/packages/core/src/stories/buttongroup.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import type { Meta } from "@storybook/react";
+
+import type { ButtonGroupProps } from "../button";
+import { ButtonGroup } from "../button";
+
+export default {
+  title: "Core/Button Group",
+  component: ButtonGroup,
+} as Meta;
+
+const Template = (props: ButtonGroupProps) => <ButtonGroup {...props} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  buttons: [
+    {
```

Comment:
you should be able to extract out the common entries into an array and share that across both stories